gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -48,6 +48,13 @@ var gpii = fluid.registerNamespace("gpii");
         members: {
             sessionIndex: {} // map of userToken to session component member name, managed by gpii.lifecycleManager.sessionIndexer
         },
+        model: {
+            logonChangeInProgressFor: undefined, // will have the value of the user token
+            lastLogonChange: {
```

Comment:
Just slightly nuts because it doesn't encode what the type of the logonChange actually was (a login or logout). Add this in for completeness